heron/scheduler-core/src/java/com/twitter/heron/scheduler/SchedulerMain.java

```diff
@@ -351,6 +353,15 @@ public boolean runScheduler() {
       // TODO(mfu): timeout should read from config
       SchedulerStateManagerAdaptor adaptor = new SchedulerStateManagerAdaptor(statemgr, 5000);
 
+      // get a packed plan and schedule it
+      PackingPlans.PackingPlan serializedPackingPlan = adaptor.getPackingPlan(topology.getName());
+      if (serializedPackingPlan == null) {
+        LOG.severe("Failed to obtain a valid PackingPlan from the state manager");
```

Comment:
I can do that. When do you think topology name should be added to log messages? Do we need a "Best Practices" guide to help contributors and reviewers?